content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -26,6 +26,12 @@ Install [`kubectl`](/docs/tasks/tools/#kubectl).
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+Set `StorageVersionMigrator` and `InformerResourceVersion` feature gates to
+`true`.
+
+Enable storage version migration REST api by setting runtime config
+`storagemigration.k8s.io/v1alpha1` to `true`.
+
```

Comment:
@sftim Is there a better way to add feature gates and runtime config info? Any example I can follow?

Reply:
There's a guide at https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates but it doesn't cover specifically what you're asking.